provider/ansible/gcp_utils.py

```diff
@@ -165,9 +165,14 @@ def _credentials(self):
             self.module.fail_json(msg="Credential type '%s' not implemented" % cred_type)
 
     def _headers(self):
-        return {
-            'User-Agent': "Google-Ansible-MM-{0}".format(self.product)
-        }
+        if self.module.params.get('env_type'):
+            return {
+                'User-Agent': "Google-Ansible-MM-{0}-{1}".format(self.module.params.get('env_type'), self.product)
```

Comment:
Will probably be easier for analytics tools to parse the string with:

    .format(self.product, self.module.params.get('env_type'))

That way the format of the string is always `Google-Ansible-MM-<product><-maybe_more_stuff>` i.e., a regex like `^Google-Ansible-MM-([^-]*).*$` with match the product name whether `env_type` is set or not..

Reply:
good call!